tests/fuzzer.rs

```diff
@@ -59,6 +59,12 @@ fn issue_22_binary_with_byte_ref_size() {
     test_fuzzer_data_ok(data);
 }
 
+#[test]
```

Comment:
Could you add a hex `<integer>` field to `tests/data/xml.plist` and update the corresponding test in `src/stream/xml_reader.rs`. That way we test hex integers are being parsed correctly.

Reply:
Done!